plonky2/src/gates/base_sum.rs

```diff
@@ -32,7 +33,8 @@ impl<const B: usize> BaseSumGate<B> {
     }
 
     pub fn new_from_config<F: Field64>(config: &CircuitConfig) -> Self {
-        let num_limbs = F::BITS.min(config.num_routed_wires - Self::START_LIMBS);
+        let num_limbs =
+            log_floor(F::ORDER as usize - 1, B).min(config.num_routed_wires - Self::START_LIMBS);
```

Comment:
It might be safest to use `u64`, I don't think `usize` is guaranteed to be as large. Or we could add `assert_eq!(size_of::<usize>() == 8);`

Reply:
Good point, I changed the function to use `u64`.